listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -295,6 +295,58 @@ def user_feed(user_name: str):
     }})
 
 
+@user_timeline_event_api_bp.get('/user/<user_name>/feed/events/<event_id>')
```

Comment:
```suggestion
@user_timeline_event_api_bp.get('/user/<user_name>/feed/event/<event_id>')
```

Reply:
I don't think this is right, as all the other event-related endpoints use the plural form (including ones that affect a single event such as `feed/events/delete` and `feed/events/hide`)

Reply:
Asked on the main chat, general agreement is to keep using the plural form since we use that for all other /events/ endpoints.
Ref: https://chatlogs.metabrainz.org/libera/metabrainz/2025-04-15/?msg=5457090&page=1